.travis.yml

```diff
@@ -1,57 +0,0 @@
-language: scala
-
-services:
-  - xvfb
-
-addons:
-  chrome: stable
-  apt:
-    packages:
-      - unzip
-      - dbus-x11
```

Comment:
Very nice the GHA VM's provide these packages.

Reply:
I'm not sure 😄 Just I migrated tests to [webdrivermanager](https://github.com/bonigarcia/webdrivermanager) and used a headless mode 
https://github.com/playframework/play-socket.io/blob/60a5dec8eade3c28c6b753510e11157667ce8197/src/test/scala/play/socketio/RunSocketIOTests.scala#L36

Reply:
That why now we don't need install Chrome driver manually and use X11 for tests 😄

Reply:
> That why now we don't need install Chrome driver manually and use X11 for tests 😄

https://github.com/actions/virtual-environments/blob/ubuntu20/20220508.1/images/linux/Ubuntu2004-Readme.md#browsers-and-drivers 😉